pom.xml

```diff
@@ -19,19 +19,26 @@
             <artifactId>selenium-java</artifactId>
```

Comment:
Using unstable version may cause come issues.
Download and set above 20 JDK 17 in environmental variables, then change maven compiler source and target. Version of used JDK you can set in File->Project structure->JDK

 <properties>
        <maven.compiler.source>20</maven.compiler.source>
        <maven.compiler.target>20</maven.compiler.target>

Reply:
We could use default-versioning in the future.

Reply:
Set "maven.compiler.source" and "maven.compiler.target" to JDK 17 as in comment above (it's last stable version compliant with Jenkins).